<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert Case</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CONVERTCASE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_frase">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Formato frase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_caixaBaixa">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Minúsculo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_caixaAlta">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Maiúsculo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_limpar">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>891</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Escreva ou cole seu texto aqui.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_removerEspaco">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remover Espaços</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
